src/client/game/numbers-round.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'; 
+import { shallow, mount } from 'enzyme';
+
+import { NumbersRound, __RewireAPI__ } from './numbers-round';
+
+const rewire = __RewireAPI__.__Rewire__;
```

Comment:
This is needed in more than 1 place. Could we have a utils file so that you can call:

```javascript
const { rewire, resetDependency } = normaliseRewire(__RewireAPI__);
```

If you need help just shout.